controllers/videos.js

```diff
@@ -5,21 +5,21 @@ const auth = require('../middleware/auth');
 
 //Get all videos
 router.get('/', auth, (req, res) => {
-	db.Video.find({ user: req.user })
+	db.Video.find({ userId: req.user })
 		.then(dbVideos => res.json(dbVideos))
 		.catch(err => res.json(err));
 });
 
 //Create video
 router.post('/', auth, (req, res) => {
-	const newVideo = { user: req.user, ...req.body };
+	const newVideo = { userId: req.user, ...req.body };
```

Comment:
you need to get  the userId from the request. 
const userId = req.userId; 
req.userId is the value you got from decode.id in auth.js.
you do not need to send the userId from the front-end.